reference/entry.json

```diff
@@ -1850,6 +1850,103 @@
         ]
       }
     },
+    "/places": {
+      "post": {
+        "summary": "post-place",
```

Comment:
The summary should be a human-readable name, as that is what is shown in the sidebar